pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -380,12 +389,17 @@ spec:
           sudo cp /var/run/secrets/istio/bootstrap/root-cert.pem /var/run/secrets/istio/root-cert.pem
           sudo cp /var/run/secrets/istio/bootstrap/*.env /var/lib/istio/envoy/
           sudo cp /var/run/secrets/istio/bootstrap/mesh.yaml /etc/istio/config/mesh
-          sudo sh -c 'cat /var/run/secrets/istio/bootstrap/hosts >> /etc/hosts'
+
+          # don't overwrite /etc/hosts since it's managed by kubeproxy
+          #sudo sh -c 'cat /var/run/secrets/istio/bootstrap/hosts >> /etc/hosts'
```

Comment:
nit: This technically drops our coverage since we don't know that the command generates the proper `IstioHosts`.

Maybe instead of writing it, we can just assert its the same as what we added in `hostAliases`? Like

```
for IstioHosts {
  grep /etc/hosts IstioHosts
}
```

or similar is probably enough..